components/clp-package-utils/clp_package_utils/scripts/compress.py

```diff
@@ -18,15 +18,8 @@
     validate_and_load_db_credentials_file,
 )
 
-# Setup logging
 # Create logger
```

Comment:
```suggestion
```

Don't think this comment is necessary. Can delete it in all the refactored files.